src/cimplog-rdk-logger.c

```diff
@@ -83,3 +83,58 @@ const char *rdk_logger_module_fetch(void)
     return NULL;
 }
 
+/*
+*
+* API to log different component logs in a same log file
+*/
+
+void __cimplog_generic(const char *rdk_logger_module, const char *module, int level, const char *msg, ...)
```

Comment:
Add generic version of this to cimplog.c too to just use printf instead of rdk logger

Reply:
As suggested by @schmidtw, code in these 2 functions should be re-used and not duplicated.